### ISSUE TITLE
document how to run lj.rpy.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5618]
|<img alt="shruti's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| shruti reported|
|-|-|
|Trac ID|trac#5618|
|Type|enhancement|
|Created|2012-04-06 15:06:23Z|

An rpy script is a file which defines a python resource and can be loaded in a Twisted Web server. These files in the repository are saved with extensions .rpy.py. These files cannot be run similar to normal python sources. I have added the documentation which would guide the users to run these files.

Attachments:

* [5618.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/2e5/2e5409786d5b375acb105b78ca5ffeacf87a728c/1b4af35b1e9112a4d93c07143ea8f0aeb14ce00c.diff) (1713 bytes) - added by shruti on 2012-04-06 18:11:59Z - The lj.rpy.py and fortune.rpy.py files now contains instructions on how to run them.
* [5618.2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/2e5/2e5409786d5b375acb105b78ca5ffeacf87a728c/88aee3873145c8b87afa4017b32f8139d5b86dac.diff) (1353 bytes) - added by shruti on 2012-04-07 16:49:28Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5618 5618
type__enhancement enhancement
reporter__shruti shruti
priority__normal normal
milestone__ 
branch__ 
branch_author__shruti shruti
status__closed closed
resolution__duplicate duplicate
component__web web
keywords__documentation documentation
time__1333724783000000 1333724783000000
changetime__1361633281000000 1361633281000000
version__None None
owner__ 
cc__jesstess cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="shruti's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|shruti set owner to shruti|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="shruti's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|shruti removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to shruti|
|-|-|

Thanks for your patch.

 - `doc/web/examples/fortune.rpy.py`
   - needs a copyright header
   - this is a minor change, so an empty .misc file is ok

- it looks like there's a word missing behind ```replacing `myusername` with a valid LiveJournal username, to see links to recent posts by that``` (user?)

**Comment by twisted-trac** [comment]
|<img alt="shruti's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|shruti removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

This was fixed in [#5612](https://github.com/twisted/twisted/issues/5612) (r34258), marking it as a duplicate.
